Auth/OpenID/Parse.php

```diff
@@ -218,12 +218,12 @@ function removeQuotes($str)
     
     function match($regexp, $text, &$match)
     {
-        if (!is_callable('mb_ereg_search_init')) {
-            if (!preg_match($regexp, $text, $match)) {
-                return false;
-            }
-            $match = $match[0];
```

Comment:
I'm concerned with removing this. [preg_match returns an array](http://php.net/manual/en/function.preg-match.php) the first element is the whole line, the second element is the text capture from `$regexp`. The purpose of this method is to just return the first line.